L4D2/Change Log.txt

```diff
@@ -1,3 +1,25 @@
+/===============================================/
```

Comment:
1.7 was never a version that got officially released for 1.7.1 to be a thing. Could just merge the two. (Also not like versions really matter now. Only really mattered before the splitter was officially added to Livesplit)

Reply:
Fixed.